doc/recipe/preprocessor.rst

```diff
@@ -180,22 +180,68 @@ To get an overview on data fixes and how to implement new ones, please go to
 
 Fx variables as cell measures or ancillary variables
 ====================================================
-The following preprocessor may require the use of ``fx_variables`` 
-to be able to perform the computations:
+The following preprocessor may require the use of ``fx_variables`` to be able
+to perform the computations:
 
-    - ``area_statistics``
-    - ``mask_landsea``
-    - ``mask_landseaice``
-    - ``volume_statistics``
-    - ``weighting_landsea_fraction``
+    - area_statistics_ (``areacella``, ``areacello``)
+    - :ref:`mask_landsea<land/sea/ice masking>` (``sftlf``, ``sftof``)
+    - :ref:`mask_landseaice<ice masking>` (``sftgif``)
+    - volume_statistics_ (``volcello``)
+    - :ref:`weighting_landsea_fraction<land/sea fraction weighting>` (``sftlf``, ``sftof``)
+
+If no ``fx_variables`` are specified for these preprocessors, the fx variables
+given in brackets are used. If given, the ``fx_variables`` argument specifies
+the fx variables that the user wishes to input to the corresponding
+preprocessor function; the user may specify it calling the variables e.g.
+
+.. code-block:: yaml
+
+    fx_variables:
+      areacello:
+      volcello:
+
+or calling the variables and adding specific variable parameters (the key-value
+pair may be as specific as a CMOR variable can permit):
+
+.. code-block:: yaml
+
+    fx_variables:
+      areacello:
+        mip: Omon
+      volcello:
+        mip: fx
```

Comment:
```suggestion
      areacello:
        mip: Ofx
      volcello:
        mip: Omon
```
The area of ocean cells typically stays the same, but changes in sea surface height vary the thickness and thus the volume.